variables.tf

```diff
@@ -94,3 +94,9 @@ variable ssl_certificates {
   description = "SSL cert self_link list. Required if `ssl` is `true` and no `private_key` and `certificate` is provided."
   default     = []
 }
+
+variable security_policy {
+  type        = "string"
```

Comment:
You can omit this as the type will be inferred by the default empty string.

Reply:
Removed.